packages/react-components/theme-designer/src/components/ColorTokens/AccessiblePairs.ts

```diff
@@ -7,98 +7,130 @@ export type AccessiblePairs = Record<string, [string, number][]>;
  * For components that need to be tested against a transparent background, only colorNeutralBackground1 is tested.
  */
 export const accessiblePairs: AccessiblePairs = {
-  colorNeutralForeground2BrandHover: [
-    ['colorTransparentBackgroundHover', 4.5],
-    ['colorSubtleBackgroundHover', 3],
-  ],
-  colorNeutralForeground2BrandPressed: [
-    ['colorTransparentBackgroundPressed', 4.5],
-    ['colorSubtleBackgroundPressed', 3],
-  ],
-  colorNeutralForeground2BrandSelected: [
-    ['colorTransparentBackgroundSelected', 4.5],
-    ['colorSubtleBackgroundSelected', 3],
-  ],
-  colorNeutralForeground3BrandHover: [],
-  colorNeutralForeground3BrandPressed: [],
-  colorNeutralForeground3BrandSelected: [],
-  colorBrandForegroundLink: [['colorNeutralBackground1', 4.5]],
-  colorBrandForegroundLinkHover: [['colorNeutralBackground1', 4.5]],
-  colorBrandForegroundLinkPressed: [['colorNeutralBackground1', 4.5]],
-  colorBrandForegroundLinkSelected: [['colorNeutralBackground1', 4.5]],
-  colorCompoundBrandForeground1: [
-    ['colorNeutralBackground1', 3],
-    ['colorTransparentBackground', 3],
-    ['colorSubtleBackground', 3],
-  ],
-  colorCompoundBrandForeground1Hover: [
-    ['colorNeutralBackground1', 3],
-    ['colorNeutralBackground1Hover', 3],
-    ['colorTransparentBackgroundHover', 3],
-    ['colorSubtleBackgroundHover', 3],
-  ],
-  colorCompoundBrandForeground1Pressed: [
-    ['colorNeutralBackground1', 3],
-    ['colorNeutralBackground1Pressed', 3],
-    ['colorTransparentBackgroundPressed', 3],
-    ['colorSubtleBackgroundPressed', 3],
-  ],
+  //   colorNeutralForeground2BrandHover: [
+  //     ['colorTransparentBackgroundHover', 4.5],
+  //     ['colorSubtleBackgroundHover', 3],
+  //   ],
```

Comment:
Were these meant to be commented out?

Reply:
Yes we'll keep these commented out for now. Someone will need go through them later and add back in if necessary. Will make a note for that.